rskj-core/src/main/java/co/rsk/core/bc/PendingStateImpl.java

```diff
@@ -404,10 +400,6 @@ private long getCurrentTimeInSeconds() {
     }
 
     private long getCurrentBestBlockNumber() {
-        if (bestBlock == null) {
-            bestBlock = this.blockChain.getBestBlock();
-        }
-
         if (bestBlock == null) {
```

Comment:
It could be better to have bestBlock not null, WHEN the pending state starts to work.

Reply:
As we spoke offline, I removed this